src/components/UserDetail/UserDetail.test.js

```diff
@@ -7,17 +7,24 @@ import Loans from './Loans';
 import UserDetail from './UserDetail';
 
 jest.mock('./Loans', () => jest.fn(() => <div />));
-jest.mock('../../../icons/user-placeholder.png', () => jest.fn(() => 'image_path'));
 jest.mock('../../util', () => ({
   getFullName: jest.fn((user) => user.personal.lastName),
 }));
+jest.mock('@folio/stripes/smart-components', () => ({
+  ProfilePicture: () => <div>ProfilePicture</div>,
+}));
```

Comment:
```suggestion
jest.mock('@folio/stripes/smart-components', () => ({
  ...jest.requireActual('@folio/stripes/smart-components'),
  ProfilePicture: () => <div>ProfilePicture</div>,
}));
```

Reply:
Moved the ProfilePicture mock to stripesSmartComponents.mock in 'test/jest/__mock__'